sdk/lib/_http/http.dart

```diff
@@ -735,6 +735,19 @@ abstract class ContentType implements HeaderValue {
   String? get charset;
 }
 
+/// Type of [Cookie.sameSite], which defines whether the cookie is available
```

Comment:
"Type" -> "Value"

(Class documentation should describe instances, not the class itself.)